rskj-core/src/main/java/co/rsk/peg/BridgeStorageProvider.java

```diff
@@ -460,21 +461,28 @@ public void setPendingFederation(PendingFederation federation) {
     /**
      * Save the pending federation
      */
-    public void savePendingFederation() {
+    public void savePendingFederation() throws IOException {
```

Comment:
Let's make this method `protected`. It's only called in this class and from tests